brew/brew-update-service-install

```diff
@@ -8,6 +8,7 @@ main() { (
 
     env PATH="$PATH" serviceman add --user \
         --name sh.brew.updater -- \
+        --workdir ~/.local/opt/brew/ \
```

Comment:
this needs to move up one line

```diff
+        --workdir ~/.local/opt/brew/ \
          --name sh.brew.updater -- \
-        --workdir ~/.local/opt/brew/ \
```

The `--` means "don't accept flags after me"